extensions/resource-deployment/src/ui/deployAzureSQLDBWizard/pages/azureSettingsPage.ts

```diff
@@ -293,9 +284,9 @@ export class AzureSettingsPage extends BasePage {
 	}
 
 	private async createServerDropdown(view: azdata.ModelView) {
-		this._serverGroupDropdown = view.modelBuilder.dropDown().withProperties({
-			required: true,
-		}).component();
+		this._serverGroupDropdown = view.modelBuilder.dropDown().withProperties({ required: true })
```

Comment:
I think just making this required should work. You do not need to add the validation part.

Reply:
There is a mock value that is provided to serverGroupDropdown saying there is no server available for the subscription that results in the required condition being fulfilled, so it is necessary to have validation for it.

Reply:
@ranasaria @aasimkhan30 For the dropdowns, I have a value that is put into the dropdown that displays a message to the user to pick a different dependent dropdown value, it causes the requirement to be fulfilled if it shows up, hence, the validation is necessary unless we want the dropdown to be blank.

Reply:
It is still not very clear as to what behavior you are trying to put in here. Perhaps pictures would help.

Reply:
I chatted with Alex offline to help with a suggestion on how to do this with example code in modelviewutils which does the kind of validation that is needed here.